packages/hub/utils/aws-config.ts

```diff
@@ -16,7 +16,10 @@ export default async function awsConfig({ roleChain = [] }) {
   let accessKeyId = config.get('aws.config.credentials.AccessKeyId');
   let secretAccessKey = config.get('aws.config.credentials.SecretAccessKey');
   if (accessKeyId || secretAccessKey) {
-    result.credentials = config.get('aws.config.credentials');
+    result.credentials = {
```

Comment:
This change fixes 2 bugs:

1. When testing locally, AWS failed to upload, returning a cryptic error `aws TypeError: Cannot read property 'byteLength' of undefined`

I discovered this is happening because the properties from the config, `AccessKeyId` and `SecretAccessKey`, should be given as `accessKeyId` and `secretAccessKey` while providing credentials to the call ([credentials should not be capitalized](https://github.com/aws/aws-sdk-js-v3/issues/2411#issuecomment-864127937)).

After changing to camelCase, the upload call started working.

2. We were attempting to write to a frozen object

Since `config.get('aws.config.credentials')` returns a frozen object, there was an exception thrown when the function [attempted to write](https://github.com/cardstack/cardstack/blob/9356b70c52e63d267b6a661b71d346c59c9c9a03/packages/hub/utils/aws-config.ts#L34)  to `result.credentials` from the `assumedRole`. The fix is to use a mutable object.